mirage/config.ts

```diff
@@ -29,29 +29,28 @@ export default function(this: Server) {
 
     this.get('/institutions');
 
-    osfResource(this, 'nodes');
-    osfNestedResource(this, 'nodes', 'contributors');
-    osfNestedResource(this, 'nodes', 'linkedNodes', { only: ['index'] });
-    osfNestedResource(this, 'nodes', 'registrations', { only: ['index'] });
-    osfNestedResource(this, 'nodes', 'draftRegistrations', { only: ['index'] });
+    osfResource(this, 'node');
+    osfNestedResource(this, 'node', 'contributors');
+    osfNestedResource(this, 'node', 'linkedNodes', { only: ['index'] });
+    osfNestedResource(this, 'node', 'registrations', { only: ['index'] });
+    osfNestedResource(this, 'node', 'draftRegistrations', { only: ['index'] });
 
-    osfResource(this, 'registrationSchemas', { path: '/schemas/registrations' });
+    osfResource(this, 'registration-schema', { path: '/schemas/registrations' });
 
-    osfResource(this, 'scopes', { only: ['index', 'show'] });
+    osfResource(this, 'scope', { only: ['index', 'show'] });
 
     this.get('/status', () => {
         return { meta: { version: '2.8' }, maintenance: null };
     });
 
-    osfResource(this, 'tokens', { except: ['create'] });
+    osfResource(this, 'token', { except: ['create'] });
     this.post('/tokens', createToken);
 
-    osfResource(this, 'users', { except: ['create', 'delete'] });
-    osfNestedResource(this, 'users', 'institutions', { only: ['index'] });
-    osfNestedResource(this, 'users', 'emails', { path: '/users/:parentID/settings/emails' });
```

Comment:
No more emails?

Reply:
Yeah, because the `user-email` model isn't in develop yet, this one couldn't typecheck.

Reply:
Or, more accurately, the `user.emails` relationship